test/modules/paymentProcessor/interfaces/IEverClearSpoke.sol

```diff
@@ -0,0 +1,116 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ * @title IEverclearSpoke
+ * @notice Interface for the Everclear spoke contract
+ */
+interface IEverclearSpoke {
```

Comment:
I think this one could be removed right? We can import the everclear interface from the src folder if we need it for testing

Reply:
Can be done in the next PR